pkg/kubelet/rkt/rkt.go

```diff
@@ -1082,7 +1082,7 @@ func (r *runtime) GetContainerLogs(pod *api.Pod, containerID string, logOptions
 }
 
 // GarbageCollect collects the pods/containers. TODO(yifan): Enforce the gc policy.
-func (r *runtime) GarbageCollect() error {
+func (r *runtime) GarbageCollect(gcPolicy kubecontainer.ContainerGCPolicy) error {
```

Comment:
So if #14686 gets in first, then the change we need to make is to derive the `--grace-period` and `--expire-prepared` from the `gcPolicy.MinAge`. AFAICS we can just set both the the `gcPolicy.MinAge`